packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/summary.css

```diff
@@ -1,13 +1,13 @@
 .root {
     display: grid;
     gap: 2rem;
-    padding: 2rem 1rem;
+    padding: 1.5rem 1rem;
 }
 
 .heading_container {
     display: grid;
     grid-auto-flow: column;
-    grid-template-columns: auto 5rem;
+    justify-content: space-between;
```

Comment:
Generally I prefer setting the first track to `1fr` rather than justifying with `space-between`.

```stylus
.foo
  // spaces 2 elements correctly, but not 3+
  // xx-----y
  // xx--y--z
  justify-content: space-between

.bar
  // spaces n elements correctly
  // xx-----y
  // xx----yz
  grid-template-columns: 1fr
```

Reply:
Ah fair - I was copying the style from the methods section to the others but I can make this change to all three :)